ci/setup-build-test-gdk.ps1

```diff
@@ -53,15 +53,8 @@ if (Test-Path env:TEST_REPO_BRANCH) {
     $test_repo_branch = $env:TEST_REPO_BRANCH
 }
 
-if (Test-Path env:GDK_SETTINGS) {
-    $user_gdk_settings = ";" + $env:GDK_SETTINGS
```

Comment:
Why this go?

Reply:
driveby, not needed, as the parsing logic already deals with it